src/community/blocks.tsx

```diff
@@ -0,0 +1,277 @@
+    title,
+    stackingGroup,
+    description,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            <Inline space="between" alignItems="center">
+                {title && (
+                    <Box paddingRight={32}>
+                        <Text2 regular>{title}</Text2>
+                    </Box>
+                )}
+                {stackingGroup ? (
+                    stackingGroup
+                ) : (
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {description}
+                    </Text2>
+                )}
+            </Inline>
+        </div>
+    );
+};
+
+interface SimpleBlockProps {
+    image?: RendersNullableElement<typeof Image>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const SimpleBlock: React.FC<SimpleBlockProps> = ({image, description, 'aria-label': ariaLabel}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            <Inline space={16} alignItems="center">
+                {image}
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {description}
+                </Text2>
+            </Inline>
+        </div>
+    );
+};
+
+interface InformationBlockProps {
+    title?: string;
+    description?: Array<string>;
+    value?: number;
+    secondaryValue?: number;
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const InformationBlock: React.FC<InformationBlockProps> = ({
+    title,
+    description,
+    secondaryValue,
+    value,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <Inline space="between" alignItems="flex-end" aria-label={ariaLabel}>
+            <BlockContent title={title} description={description} />
+            <div className={classNames(styles.column, styles.rightContent)}>
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {secondaryValue}
+                </Text2>
+                <Text5 color={valueColor}>{value}</Text5>
+            </div>
+        </Inline>
+    );
+};
+
+interface HighlightedValueBlockProps {
+    headline?: RendersNullableElement<typeof Tag>;
+
+    mainHeading: {
+        value: number;
+        text: string;
+    };
+
+    secondHeading?: {
+        value: number;
+        text: string;
+    };
+
+    secondaryValue?: number;
+
+    title?: string;
+    description?: Array<string>;
+
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const HighlightedValueBlock: React.FC<HighlightedValueBlockProps> = ({
+    headline,
+    mainHeading,
+    secondHeading,
+    secondaryValue,
+    title,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            {headline && <Box paddingBottom={24}>{headline}</Box>}
+
+            <Stack space={2}>
```

Comment:
Is this component the one to render this?

![image](https://github.com/Telefonica/mistica-web/assets/1775361/cab0414d-96c0-4516-9493-e89d5fefa1c2)

For first slot:
I assume 300 Mega => would be mainHeading.value
I assume de velocidade => would be mainHeading.text

But there's no possibility of putting a text above this mainHeading for "Vivo Fibra 300 Mbps Especial Netflix_1"
As headline is a Tag.

----------------------

For second slot, similar problem:

I assume 110 canais is => mainHeading.value

But mainHeading.text is mandatory.
And there's no possibility of putting a text above this mainHeading for "Vivo Play Avançado HD"
As headline is a Tag.

**************

The other alternative is that we are supposed to use the ProgressBlock for this but without a progress bar?

Note: screenshot tests for the actual cases we want to use would be helpful in identifying which block to be used in each case.

Reply:
@BrunoGuimaraes103, this is important. Can you check the definition with the designers? The blocks should be able to cover these cases

Reply:
We are checking about it

Reply:
@Winde Sorry, I'm kind of confused on what you mean, could you be a little clearer?

The use cases in question can be done using ProgressBlock and ValueBlock as shown in this image:
![image](https://github.com/Telefonica/mistica-web/assets/123959276/ff2cefd7-a32b-4271-a149-731e74a3517c)

Reply:
It's a bit weird:
(1) to have to use a Progress block without progress bar
(2) That these two cases are actually different blocks

but fair enough :)

Reply:
yes, it's quite weird. I said the same in other comment :(